pose_evaluation/examples/example_metric_construction.py

```diff
@@ -0,0 +1,75 @@
+from pathlib import Path
+
+from pose_format import Pose
+
+from pose_evaluation.metrics.distance_metric import DistanceMetric
+from pose_evaluation.metrics.distance_measure import AggregatedPowerDistance
+from pose_evaluation.metrics.base import BaseMetric
+from pose_evaluation.metrics.test_distance_metric import get_poses
+# from pose_evaluation.metrics.pose_processors import get_standard_pose_processors
+# from pose_evaluation.metrics.dynamic_time_warping_metric import DTWMetric
+
+if __name__ == "__main__":
+
+    reference_file = Path(r"pose_evaluation\utils\test\test_data\colin-1-HOUSE.pose")
```

Comment:
paths with backslash? are you on windows? that would not work on linux/mac, right?